.github/actions/java-test/action.yaml

```diff
@@ -35,9 +35,17 @@ runs:
     - name: Run Maven compile
       shell: bash
       run: |
-        ./mvnw compile test-compile scalafix:scalafix -Psemanticdb
+        if [ $JAVA_VERSION == "8" ]; then
+          ./mvnw -B compile test-compile scalafix:scalafix -Psemanticdb -Djava.version=1.8
+        else
+          ./mvnw -B compile test-compile scalafix:scalafix -Psemanticdb -Djava.version=${JAVA_VERSION}
+        fi
 
     - name: Run tests
       shell: bash
       run: |
-        SPARK_HOME=`pwd` ./mvnw clean install
+        if [ $JAVA_VERSION == "8" ]; then
+          SPARK_HOME=`pwd` ./mvnw -B clean install -Djava.version=1.8
```

Comment:
can we just set `JAVA_VERSION` to `1.8` so we don't need this if-else statement?